Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -123,6 +125,19 @@ final class StatusTableViewController: LoopChartsTableViewController {
         addScenarioStepGestureRecognizers()
 
         tableView.backgroundColor = .secondarySystemBackground
+    
+        tableView.register(TempStatusCell.self, forCellReuseIdentifier: TempStatusCell.className)
+        notificationsCriticalAlertPermissionsViewModel.$showWarning
+            .receive(on: RunLoop.main)
+            .sink { [weak self] in
```

Comment:
[nit] Since this is more than a one line function, please use an explicit local variable name (not `$0`), such as `showWarning`.